### PR TITLE
extend german vocabulary in words.json

### PR DESCRIPTION
**Pull request by jonathan-scholbach**

### CODE REVIEW

**Review by alexhans1 — CHANGES_REQUESTED** [review]
there are about 25 duplicate words now. Here's the sorted German word list without duplicates: 
```
"German": "ABFLUSS, ABKRATZEN, ABLEHNUNG, ABSTRAKT, ABSTURZ, AGENT, ÄHRE, AKNE, AKTIONÄR, ALBTRAUM, ALIEN, ALIMENTE, ALKOHOL, ALLE, ALLIGATOR, ALPHATIER, ALPTRAUM, ALT, AMERIKA, ANGELN, ANKER, ANWENDUNG, APATHIE, APFELSAFT, APOTHEKE, APPLAUS, ARBEIT, ARBEITSLOS, ARCHÄOLOGE, ARISTOKRAT, ARM, ARMADA, ARMBANDUHR, ART, ARZT, ASCHE, ASTRONAUT, ATLANTIS, ATMEN, AUFGABE, AUFTAUEN, AUFZUG, AUGENBRAUE, AUSBLENDEN, AUSGEFALLEN, AUSSENSEITE, AUSWAHL, AUTO, AUTOBAHN, AVOCADO, BABY-SITTER, BAGUETTE, BALKON, BALLON, BANANE, BÄNDER, BANK, BASEBALL, BASKETBALL, BATTERIE, BAUER, BAUM, BEDAUERN, BEDIENEN, BEETHOVEN, BEGRIFF, BEIN, BEITRAG, BENZIN, BERG, BESEN, BESSERWISSER, BETRIEB, BETRUG, BETTWANZE, BEUTEL, BEVÖLKERUNG, BIER, BILD, BIRNE, BISS, BLASE, BLATT, BLAU, BLAUPAUSE, BLAUWAL, BLEICHMITTEL, BLEISTIFT, BLINZELN, BLUME, BLÜTEZEIT, BOBFAHREN, BODEN, BOHNENSTANGE, BOMBE, BONSAI, BOOT, BOX, BRATEN, BRAUCHTUM, BRAUT, BRÄUTIGAM, BRAUTKLEID, BREMSLICHT, BRIEF, BRILLE, BROKKOLI, BROTZEIT, BRÜCKE, BRUDER, BRÜNETT, BRUNFT, BRUST, BRUT, BUCH, BUCHEN, BÜFFEL, BUG, BUNTSTIFT, BÜRO, BUS, BUTTERBROT, CAFETERIA, CAMPINGPLATZ, CD, CHAMPION, CHARME, CHEERLEADER, CHEF, CHEFKOCH, CHINA, CLOWN, COACH, COMPUTER, COWBOY, CREME, DACH, DAME, DANEBEN, DANKBARKEIT, DART, DECKE, DELLE, DEMONTIEREN, DEPONIE, DIEB, DIELE, DIREKTOR, DOMINOSTEINE, DONNER, DOSE, DRACHEN, DREIECK, DRÜCKEN, DUNST, DUSCHE, DUZENT, EBENHOLZ, ECKE, EDELSTEIN, EHRE, EIDECHSE, EIMER, EINBRUCH, EINLADUNG, EINS, EINZELN, EIS, EISBAHN, EISEN, EISTEE, EIZELLE, ELEFANT, ELEKTRIZITÄT, ELEMENT, ELF, ELFENBEIN, ELLENBOGEN, ENDLOS, ENERGIE, ENGLAND, ENTE, ENTWICKLUNG, ERBEN, ERDE, ERGONOMISCH, ERSCHROCKEN, ERSTATTUNG, ERWEITERUNG, ETAPPE, EUREKA, EUROPA, FAHNENMAST, FAHRRAD, FALTE, FÄSSER, FASTENBRECHEN, FATAL, FAULENZER, FEE, FEHLER, FELGE, FERNSEHEN, FERTIG, FEST, FEUDALISMUS, FEUER, FINGER, FIX, FLACH, FLANELL, FLATTERN, FLEDERMAUS, FLIEGE, FLOSS, FLUGZEUGE, FOLIE, FORM, FOTO, FRACHT, FRANKREICH, FRANSEN, FRAU, FREUDE, FREUNDLICHKEIT, FREUNDSCHAFT, FRIEDEN, FROSCH, FRÜHLING, FUNKEN, FUSSBALL, GALOPP, GANG, GANZ, GARTEN, GAS, GASTHAUS, GEBÄCK, GEBEN, GEBIET, GEBROCHEN, GEBURTSTAG, GEDANKE, GEDICHT, GEFAHR, GEGENTEIL, GEHEIM, GEIGE, GELÄNDER, GELD, GELEE, GEMEIN, GENAU, GENOSSE, GERICHT, GESCHIRR, GESELLSCHAFT, GESICHT, GESINNUNG, GEWEBE, GIESSKANNE, GIFT, GIPFEL, GITARRE, GLANZ, GLOCKENSPIEL, GLÜCKLICH, GLÜHBIRNE, GOLD, GOTT, GRABEN, GRAFIK, GRAPHIT, GRAS, GRAU, GREIFEN, GRIFF, GRIPPE, GROSS, GRÜBCHEN, GRUBE, GRÜN, GRUND, GRUPPE, GUMMI, GÜRTEL, HAARE, HAFERMILCH, HAFTEN, HAI, HALB, HAMMER, HANDELN, HANDSCHRIFT, HANDSCHUH, HANDY, HÄNGEN, HASE, HAUS, HAUSAUFGABEN, HAUSBOOT, HECK, HECKE, HEFT, HEIM, HEISS, HEMD, HERAUSGEBER, HERBERGE, HERD, HERZ, HIMMEL, HINWEIS, HIRTE, HOCH, HOCHZEIT, HOCKEY, HORST, HOSEN, HOTEL, HUBSCHRAUBER, HÜGEL, HUHN, HUND, HUNDEHÜTTE, HUNGRIG, HUPEN, HÜRDE, HUSTEN, HUT, HÜTTE, ILLEGAL, IMPLODIEREN, INDIANER, INGWER, INQUISITION, INTERNET, IRONIE, JAGUAR, JAHRHUNDERT, JAPAN, JEANS, JET, JOGGEN, JOURNAL, JUGENDLICHE, JUNGE, KABINE, KAHL, KAISER, KALK, KALT, KANADA, KANONE, KANTE, KANZLER, KAPITALISMUS, KARTOFFEL, KARTOGRAPHIE, KARTON, KATZE, KAUEN, KAUFEN, KAUGUMMI, KEGEL, KERZE, KETTE, KILLER, KILOGRAMM, KINDERBETT, KIRCHE, KISSEN, KITA, KLAPPE, KLAVIER, KLEID, KLEIDERSTANGE, KLEMPNER, KLIPPE, KLUFT, KNALLHART, KNEIFEN, KNETE, KNIE, KNÖCHEL, KNUSPRIG, KOALA, KOCHEN, KOHLE, KOMFORT, KOMMENTAR, KOMPASS, KÖNIG, KÖNIGIN, KONTINUUM, KONVERSATION, KOPFHÖRER, KÖRPER, KORRIGIEREN, KRÄHE, KRANK, KRAUT, KREIDE, KREISVERKEHR, KREUZFAHRT, KRIPPE, KRITISIEREN, KRÜMEL, KRUSTE, KÜCHE, KUCHEN, KUH, KULI, KULTUR, KUNST, KURZ, KUSCHELN, LACHEN, LACHS, LACK, LADEGERÄT, LAKEN, LANGSAM, LÄRCHE, LAST, LASTER, LATEIN, LÄTZCHEN, LAUTSPRECHER, LEBENSSTIL, LEBERFLECK, LEBKUCHEN, LECKEN, LEHRER, LEIDENSCHAFT, LEISE, LEITER, LEUCHTE, LEVEL, LICHT, LICHTSCHWERT, LIEBE, LIED, LIEDTEXTE, LINIE, LOCH, LÖFFEL, LOLLI, LOS, LÖSCHEN, LÖWE, LOYALITÄT, LUFT, LUFTSCHIFF, LUMPEN, LUNCHBOX, LUST, MACHO, MÄDCHEN, MÄHER, MAMA, MAMMUT, MANN, MANSCHETTE, MARIA, MARIENKÄFER, MARIONETTE, MARKE, MARKIERUNG, MARS, MÄRTYRER, MASCHINE, MASKOTTCHEN, MASSSTAB, MAST, MATRATZE, MAUER, MAUS, MESSER, MEXIKO, MINE, MINUS, MISTGABEL, MITTAGESSEN, MITTEL, MITTSOMMER, MODERN, MOMENT, MOND, MONITOR, MONSTER, MONTAG, MOOS, MORAL, MORGENROT, MOTOR, MOTORHAUBE, MOTORRAD, MÜDE, MUMMENSCHANZ, MURMEL, MUTTER, NACHT, NAGEL, NAIV, NARBE, NASE, NATION, NATUR, NEBEL, NEHMEN, NEST, NETZ, NEUTRON, NICHTE, NICHTSNUTZ, NIESEN, NONNE, NOTLÜGE, NULL, NUTZEN, OFEN, OFFEN, ÖFFNER, OHR, ÖL, OLYMPIONIKE, OPA, ORGAN, ORGANISIEREN, ORT, OSTERN, OZEAN, PALAST, PAPA, PAPIER, PARK, PARODIE, PARTY, PASSWORT, PAUKEN, PENDEL, PENIS, PENNY, PERSÖNLICH, PERÜCKE, PFEFFER, PFEIFEN, PFERD, PFLANZE, PFLEGE, PFLOCK, PFLÜGEN, PHILOSOPH, PHÖNIX, PICKNICK, PILOT, PILZ, PING, PIRAT, PLAKATWAND, PLANEN, POL, POMP, PONG, POOL, PORTFOLIO, POSITIV, POSTFACH, PRAKTIKANT, PRELLUNG, PRIESTER, PRINZESSIN, PRIVAT, PROKURIST, PROTESTANT, PROTHESE, PROTOKOLL, PSYCHOLOGIN, PULLOVER, PUNK, PUNKT, PUPPE, PYJAMAS, QUADRATISCH, QUARANTÄNE, RADIO, RASEN, RASENMÄHER, RÄTSEL, RAUM, RECHNER, RECHT, RECYCELN, REDNER, REGAL, REGEN, REGENBOGEN, REGENWASSER, REIHE, REIM, REIS, REISE, REITEN, RELATION, RENNEN, RICHTIG, RING, RIPPE, ROLLE, ROLLSTUHL, ROLLTREPPE, ROSE, ROSINE, ROT, RÜCKGRAT, RUCKSACK, RUDER, RUND, RUNDE, SAISON, SAITE, SALON, SALZ, SAMMLUNG, SANDBURG, SANDKASTEN, SANDWICH, SÄNGER, SATELLIT, SATZ, SAUM, SCHACH, SCHADEN, SCHAFE, SCHÄRPE, SCHATZ, SCHAUFEL, SCHEUERN, SCHIFF, SCHIFFBRUCH, SCHIMPANSE, SCHIRM, SCHLAF, SCHLAFEN, SCHLAMM, SCHLAUCH, SCHLEIER, SCHLEIFE, SCHLENDRIAN, SCHLIPS, SCHLUCK, SCHLUMMERN, SCHLÜSSEL, SCHMUTZIG, SCHNABELTIER, SCHNEE, SCHNEEBESEN, SCHNELL, SCHNUR, SCHNÜRSENKEL, SCHOKOLADE, SCHÖNHEIT, SCHONZEIT, SCHRAUBE, SCHREIBTISCH, SCHRUMPFEN, SCHULE, SCHURKE, SCHWAMM, SCHWARM, SCHWEISSER, SCHWIEGERMUTTER, SCHWIMMEN, SCHWINDEL, SCHWINGEN, SEGEL, SEITE, SET, SHAMPOO, SHERIFF, SICHER, SICHERHEIT, SICHTWEISE, SIEB, SIESTA, SIGNAL, SILHOUETTE, SINN, SKI, SOFA, SOFTDRINK, SONNE, SONNENBRAND, SOUVENIR, SOZIALISMUS, SPEED, SPIEGEL, SPIEL, SPIELPLATZ, SPINNE, SPIONIEREN, SPITZE, SPORTLER, SPRINGER, SPROSSE, SPRUDELN, SPRUNG, SPUCKE, SPULE, SPÜLUNG, STAAT, STAB, STACHEL, STAMM, STAND, STARK, STAUB, STECKEN, STEHEND, STEIN, STEPPE, STERN, STERNWARTE, STIFT, STIMME, STIRN, STIRNRUNZELN, STOFF, STRAND, STREAM, STREICHHOLZ, STREIFEN, STROHHALM, STUDENT, STUHL, STUMM, STUMPF, STURM, SÜDEN, SUMME, SUMPF, SUPPE, SUSHI, SÜSSIGKEITEN, TAG, TAGESZEIT, TAIGA, TANTE, TASCHE, TASCHENLAMPE, TASSE, TAU, TAUCHER, TAXI, TEEKANNE, TEELÖFFEL, TEENAGER, TEIL, TELEFON, TELLER, TENNIS, TIEF, TIER, TIGER, TISCH, TOAST, TOD, TOILETTE, TON, TÖNUNG, TORNADO, TRAINIEREN, TRAKTOR, TRAUBE, TRAUER, TRAUM, TRÄUMEN, TRAURIG, TREIBEN, TREIBSAND, TREPPE, TRICHTER, TRINKEN, TROCKNER, TROLL, TROMPETE, TROPFEN, TRUMPF, TUBA, TÜR, TURNIER, TÜTENSUPPE, TUTOR, ÜBERSCHRIFT UHR, UHRWERK, UMARMUNG, UMHANG, UMLAUFBAHN, UNSCHÄRFE, UNTERSCHRIFT, UNTERSETZER, VEGAN, VENTILATOR, VENUS, VERBINDUNG, VERENGUNG, VERGELTUNG, VERHANDELN, VERLETZT, VERRÜCKT, VERSPRECHEN, VERWIRRUNG, VERZÖGERUNG, VISION, VOGEL, VORHANG, VORRICHTUNG, WACHS, WAND, WANDEL, WANNE, WÄSCHEREI, WASSER, WASSERMELONE, WEDELN, WEISS, WEIZEN, WELLE, WELPE, WELT, WERBEN, WERKZEUG, WERT, WESTE, WILLE, WINDMÜHLE, WINKEL, WINTER, WISCHMOPP, WITZ, WOLF, WOLLE, WÖRTERBUCH, WUNSCH, WURM, WURZEL, ZAHNARZT, ZAHNBÜRSTE, ZAHNRAD, ZAR, ZAUN, ZEHENSPITZEN, ZEHN, ZEICHNEN, ZEIT, ZEITUNG, ZELLE, ZEN, ZIEL, ZIRKUS, ZOLLSTOCK, ZONE, ZOO, ZUCKER, ZUFÄLLIG, ZUG, ZWEIG, ZWERG",
```
Tip: VS Code extension `tyriar.sort-lines` can sort lines and remove duplicates. You just have to replace all the spaces with a line break first.

**Comment by jbowens** [comment]
might also want to look at #39. I don't know German so I'll let y'all come to an agreement on the word list to use.

File: frontend/words.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
...HL, STUHL, STUMM, STUMPF, STURM, SUMME, SUMPF, SUPPE, SUSHI, SÄNGER, SÜDEN, SÜSSIGKEITEN, TAG, TAGESZEIT, TAIGA, TANTE, TASCHE, TASCHENLAMPE, TASSE, TAU, TAUCHER, TAXI, TEEKANNE, TEELÖFFEL, TEENAGER, TEIL, TELEFON, TELLER, TENNIS, TIEF, TIER, TIGER, TISCH, TOAST, TOD, TOILETTE, TON, TORNADO, TRAINIEREN, TRAKTOR, TRAUBE, TRAUER, TRAUM, TRAURIG, TREIBEN, TREIBSAND, TREPPE, TRICHTER, TRINKEN, TROCKNER, TROLL, TROMPETE, TROMPETE, TROPFEN, TRUMPF, TRÄUMEN, TUBA, TURNIER, TUTOR, TÖNUNG, TÜR, TÜTENSUPPE, UHR, UHR, UHRWERK, UMARMUNG, UMHANG, UMLAUFBAHN, UNSCHÄRFE, UNTERSCHRIFT, UNTERSETZER, VEGAN, VENTILATOR, VENUS, VERBINDUNG, VERENGUNG, VERGELTUNG, VERHANDELN, VERLETZT, VERRÜCKT, VERSPRECHEN, VERWIRRUNG, VERZÖGERUNG, VISION, VOGEL, VORHANG, VORRICHTUNG, WACHS, WAND, WANDEL, WANNE, WASSER, WASSERMELONE, WEDELN, WEISS, WEIZEN, WELLE, WELPE, WELT, WERBEN, WERKZEUG, WERT, WESTE, WILLE, WINDMÜHLE, WINKEL, WINTER, WISCHMOPP, WITZ, WOLF, WOLLE, WUNSCH, WURM, WURZEL, WÄSCHEREI, WÖRTERBUCH, ZAHNARZT, ZAHNBÜRSTE, ZAHNRAD, ZAR, ZAUN, ZEHENSPITZEN, ZEHN, ZEICHNEN, ZEIT, ZEIT, ZEITUNG, ZELLE, ZEN, ZIEL, ZIRKUS, ZOLLSTOCK, ZONE, ZOO, ZUCKER, ZUFÄLLIG, ZUG, ZWEIG, ZWERG, ÄHRE, ÖFFNER, ÖL, ÜBERSCHRIFT",
```
Comment (alexhans1) [inline frontend/words.json]: `UHR` is duplicated

**Review by alexhans1 — APPROVED** [review]

**Comment by alexhans1** [comment]
looks like [#39](https://github.com/jbowens/codenames/pull/39) includes all these words plus some 300 more.

**Pull request closed without merging**